tests/e2e/verify-install/keycloak/keycloak_test.go

```
@@ -658,3 +669,20 @@ func verifyUserClientRole(user, userRole string) bool {
 	}
 	return false
 }
+
+func verifyMySQLPodAnnotation() bool {
```

This should return an error to help with diagnostics, right?